packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -906,15 +973,65 @@ export function getParameters(
 }
 
 /**
- * Returns an array of server objects for the opeartion at the given path and
+ * Returns an hash containing the callbacks defined in the given endpoint.
+ */
+export function getEndpointCallbacks(
+  path: string,
+  method: string,
+  oas: Oas3,
+  data: PreprocessingData
+): { [key: string]: CallbackObject } {
+  const callbacks = {}
+  const endpoint: OperationObject = oas.paths[path][method]
```

Comment:
Just a small nitpick, but I would call this an "operation" instead of "endpoint"

Reply:
Yes probably, i reused the same terminology of above functions.